garden_ai/app/notebook.py

```diff
@@ -218,6 +235,17 @@ def start(
         source_path = top_level_dir / "notebook_templates" / template_file_name
         shutil.copy(source_path, notebook_path)
 
+    # Get base image name from notebook if user did not provide
+    if base_image_name is None:
+        # If a user is using a custom image uri, base_image_name might be None
```

Comment:
Not in scope for this PR, but this probably means that we should add support for the custom image case in the notebook metadata